README.rst

```diff
@@ -52,15 +52,15 @@ Unit Tests
 
 Twisted has a comprehensive test suite, which can be run by ``tox``::
 
-  $ tox -l            # to view all test environments
-  $ tox -e py27-tests # to run the tests for Python 2.7
-  $ tox -e py35-tests # to run the tests for Python 3.5
+  $ tox -l                       # to view all test environments
+  $ tox -e py35-tests            # to run the tests for Python 3.5
+  $ tox -e py36-tests            # to run the tests for Python 3.6
```

Comment:
I think that a single py3 is enough ... and maybe suggest python3.8 as default.

Reply:
Actually, it looks like `tests` was removed from tox in https://github.com/twisted/twisted/commit/fcd6ea08f4043996f5d886625da7a2996dda3372

I changed to:

```
+  $ tox -l                       # to view all test environments
+  $ tox -e nocov                 # to run all tests without coverage
+  $ tox -e withcov               # to run all tests for with coverage
+  $ tox -e alldeps-withcov-posix # install all dependencies, run tests on POSIX platform
```